### DISCUSSION TITLE
Non "drag and drop" feature for adding/visualizing data

### DISCUSSION THREAD

**Discussion by hansryant** [facontidavide/PlotJuggler#981] [General]
Is there any other way to add the data from the "Timeseries List" to the plotting area without the drag & drop feature? I have a problem with my machine that I can't drag & drop the data, which brings back to the original question if there is a way to add the data without having to use the "Drag & Drop"? Thank you

**Comment by facontidavide** [comment]
No, there isn't

**Comment by Bartimaeus-** [comment]
@hansryant if you are desperate for a workaround, you could create a config file that has the series you want added to the plots (I'd recommend using a machine that can do drag and drop to create the config file and then put it onto your main machine) and then you just have to load the config file. I suspect that's just the start of your problems if you can't drag & drop though.

**Reply by hansryant** [reply]
@Bartimaeus- this hack works as I wanted! thanks for the tips!
